main.1.tf

```diff
@@ -0,0 +1,110 @@
+variable "region" {
```

Comment:
Same applies, add author, documentation, help with naming conventions and navigation.